tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -972,6 +972,10 @@ def _convert(cli_parser: argparse.ArgumentParser, framework, args, python_api_us
                 if not argv.use_legacy_frontend and framework_is_tf(args, argv) and not is_fallback:
                     print(get_try_legacy_fe_message())
 
+        msg = remove_path_lines(str(e.with_traceback(None))).strip()
+        if len(msg):
+            telemetry.send_event('mo', 'error_info', "error_message:'{}'".format(msg))
```

Comment:
I am afraid that we will duplicate telemetry messages sent in FE itself and here. For example, it can happen for unsupported translators.